src/Makemodule.am

```diff
@@ -111,14 +111,15 @@ src_libczmq_la_LDFLAGS += \
     -avoid-version
 endif
 
+if ON_CYGWIN
+src_libczmq_la_LDFLAGS += \
+    -no-undefined \
+    -avoid-version
+endif
+
 src_libczmq_la_LIBADD = \
     ${zmq_LIBS}
 
-if ON_ANDROID
-src_libczmq_la_LIBADD += \
-    -llog
-endif
-
```

Comment:
Please revert this removal - this bit is important for logging to the system service on android.